### PR TITLE
Prevent MissingMethod exception with WAM on Server 2016

### PR DESCRIPTION
**Pull request by bgavrilMS**

### CODE REVIEW

File: src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/LegacyOsWamProxy.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Core;
+using Windows.Security.Authentication.Web.Core;
+using Windows.Security.Credentials;
+
+#if NET5_WIN
+#elif DESKTOP || NET_CORE
+using Microsoft.Identity.Client.Platforms;
+#endif
```
Comment (pmaytak) [inline src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/LegacyOsWamProxy.cs:14]: Can be removed.

**Review by pmaytak — APPROVED** [review]
👍

**Review by trwalke — APPROVED** [review]
LGTM

File: src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/WamExtensions.cs [hunk 2]
```diff
@@ -26,8 +26,7 @@ public static string ToLogString(this WebTokenRequest webTokenRequest, bool pii)
         {
             StringBuilder stringBuilder = new StringBuilder();
             stringBuilder.AppendLine("=== WebTokenRequest ===");
-            stringBuilder.AppendLine($"ClientId: {webTokenRequest?.ClientId}");
-            stringBuilder.AppendLine($"CorrelationId: {webTokenRequest?.CorrelationId}");
```
[the remark this answers is not in the document]
Comment (bgavrilMS) [reply src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/WamExtensions.cs:30]: It causes that error. We log the correlation ID in many other places, so I think it's fine not to log it here.

**Pull request merged**